src/credentials/examples/DeviceAttestationCredsExample.cpp

```diff
@@ -16,19 +16,13 @@
  */
 #include "DeviceAttestationCredsExample.h"
 
+#include <credentials/examples/ExampleDACs.h>
+#include <credentials/examples/ExamplePAI.h>
 #include <crypto/CHIPCryptoPAL.h>
 #include <lib/core/CHIPError.h>
 #include <lib/support/Span.h>
 
 // TODO: Remove once the Attestation Credentials storage mechanism is updated.
```

Comment:
Remove TODO?

Reply:
I don't know if that's really done - WAS the attestation credentials storage updated? I'm not sure what the goal there is. We're really not pulling any of these certs from a proper storage solution - these are compiled into the image.

Reply:
If we have a proper storage solution, we should use it, but that's outside the scope of this PR>

Reply:
I'm going to leave this for now so I don't accidentally close the todo issue. I don't think this todo is really fixed.